packages/block-library/src/latest-posts/edit.js

```diff
@@ -391,11 +401,22 @@ class LatestPostsEdit extends Component {
 							'';
 
 						const imageSourceUrl = post.featuredImageSourceUrl;
-
 						const imageClasses = classnames( {
 							'wp-block-latest-posts__featured-image': true,
 							[ `align${ featuredImageAlign }` ]: !! featuredImageAlign,
 						} );
+						const renderFeaturedImage =
+							displayFeaturedImage && imageSourceUrl;
+						const featuredImage = renderFeaturedImage && (
+							<img
+								src={ imageSourceUrl }
+								alt=""
```

Comment:
It's not from this PR but I think maybe this alt should not be empty.

Reply:
Yes! I missed that because `img` tag was there, but it should be fixed here. Thanks!

Reply:
I can see that in front-end the image is retrieved through `get_the_post_thumbnail` which echos the `alt` attribute by default ( https://core.trac.wordpress.org/browser/tags/5.5/src/wp-includes/media.php#L1038 ), so no extra work is needed there..

> There might be some SEO implications here with adding these links.

I added `alt` in the editor, but I don't think there are SEO implications since in front-end is echoing the attribute. It feels like more accessibility issue in editor, that is now addressed